src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -405,7 +400,7 @@ void func_8092D1B8(EnDns* this, GlobalContext* globalCtx) {
             gSaveContext.eventInf[1] |= 0x20;
             this->unk_2F4 = func_8092CCEC;
             func_8092C63C(this, 2);
-            this->actionFunc = func_8092D320;
+            this->actionFunc = EnDns_Noop;
```

Comment:
Sorry, didn't read this properly. This should be `EnDns_DoNothing` since it's an actionFunc.